### DISCUSSION TITLE
Need Help with GitHub Global Campus Application Process

### DISCUSSION THREAD

**Discussion by ShaileshJadav2732** [community/community#143251] [GitHub Education]
### Hi everyone,

Hi everyone! I’m new to the GitHub Education community, and I’m excited to connect and learn with all of you. 🎉

I’m currently working on my application for GitHub Global Campus, but I have some questions about the process. Specifically, I’m wondering what kind of documents are most commonly accepted for verification. I’ve tried using my university ID, but I received a rejection asking for additional proof.

If anyone has experience with successful applications or knows the types of documents that work best, I’d love to hear your advice. Thanks in advance for any tips you can share!
#50399

**Comment by davevad93** [comment]
Hi @Shaileshjadav2732 ,

As stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student), these are the requirements and documents needed to apply for the `Student Developer Pack`:


- **To be eligible for GitHub Education, including GitHub Student Developer Pack and other benefits, you must:**

    - **Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, 
      homeschool, or similar educational institution**
    - **Have a verifiable school-issued email address or upload documents that prove your current student status**
    - **Have a [GitHub personal account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github)**
    - **Be at least 13 years old**
    - **Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, t 
      transcript, and affiliation or enrollment verification letter.**

In addition to your ID, I advise you to also provide a document with a current date in a `DD/MM/YYYY`, `MM/DD/YYYY` or `YYYY/MM/DD` format, for example your class schedule or the results of your last exams.

**Comment by unusdv** [comment]
Hi! Welcome! 🎉 For verification, use documents like a tuition receipt, enrollment letter, or transcript. A university email can help too. Good luck!

**Comment by ShaileshJadav2732** [comment]
I can't access my college account so how I apply student developer pack
